src/lib.rs

```diff
@@ -352,6 +362,14 @@ impl SymmetricKey {
         }
     }
 
+    /// Create a new key from bytes.
```

Comment:
Just curious, in what cases do we store our encryption key serialized? :)

Reply:
@povilasb, the access container in the Authenticator actually stores the symmetric encryption keys in the serialised form (to be more exact, it stores the [`MDataInfo`](https://github.com/maidsafe/safe_client_libs/blob/4f8b40f53fad1ff9b874085dd10c51ad0090c42a/safe_core/src/client/mdata_info.rs) structs that can point to private containers, along with the encryption key).